baron/__init__.py

```diff
@@ -35,5 +35,5 @@
 
 async def setup(bot: Red) -> None:
     cog = Baron(bot)
-    bot.add_cog(cog)
-    await cog.build_cache()
+    await bot.add_cog(cog)
+    await cog.cog_load()
```

Comment:
This line should be removed  and `build_cache` should be renamed to `cog_load` at `baron/baron.py` at line 95

Reply:
oh thanks for pointing out didn't notice that it should have been renamed on the `baron.py` file